### ISSUE TITLE
Self-Publishing Spartacus Libraries Using the Open Source Code Support

### ISSUE THREAD

**Issue by RudlTier** [SAP/spartacus#17709]
TLDR: Can i still use the "Self-Publishing Spartacus Libraries Using the Open Source Code" approach for this project also for version 6 and newer. Can it be expected that open source support continues also in the future?

---

I hope you're doing well! I'm interested in using Spartacus in a Software Project. Cecause I do not have access to the Composable Storefront Release i wanted to use the Open Source Project. While diving deeper into the evaluation I was wondering what the support status of this project is:

Here some things that I noted that seem a bit odd:

Firstly, I noticed a little discrepancy with Version 6. The documentation suggests it hasn't been released yet, but the latest composable storefront release is already 6.2. I was wondering whether there a plans to also provide offical release branches for version 6.2 in this repository.

Another thing that piqued our curiosity is how code development happens in an open-source setting (Pull Request etc.), but ticket management seems to be handled in a private Jira tenant. Any insights into the reasoning behind this, and do you have plans to streamline these processes?

Lastly, I was trying to install from local sources, but it's proving a bit trickier than expected (https://github.com/SAP/spartacus/blob/develop/docs/self-publishing-spartacus-libraries.md) . I'm not sure if the documentation is up-to-date, so any guidance or the right resources would be fantastic.
Some things that produced trouble:
- schematics/testing.ts is only working on linux per default. Needs some custom adjustments to work on windows (remove execSync('rm -rf ./scripts/install/storage');)
- Package.json needs adjustment : "@spartacus-eslint/eslint-plugin": "link:./projects/eslint" ->"@spartacus-eslint/eslint-plugin": "file:./projects/eslint",
- Unclear which npm version to use to build the libraries

We appreciate your time in addressing these queries. We just want to make sure we're on the same page and support the project to the fullest.

**Comment by janwidmer** [comment]
Hey @RudlTier, 

You correctly noticed, that release branches are missing for the latest versions, I created a ticket for it: https://github.com/SAP/spartacus/issues/17736

Regarding Open source and code development, saidly the current state is not really ideal for us developers. Github Issues are not checked regularley and seem to have really low priority.

Regarding node versions, in the last spartacus webinar hosted by the product owner of Spartacus, the following was said about Node:
<img width="1366" alt="image" src="https://github.com/SAP/spartacus/assets/3381820/ad700098-410c-498d-8f1d-6cd69bab71b9">

At the moment, the still use Yarn instead of NPM to develop spartacus.
